src/web_app/css/main.css

```diff
@@ -104,6 +104,22 @@ footer > div.active {
   opacity: .8;
 }
 
+div#privacy {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  /* Makes sure to stay behind room links so they can be clicked */
+  z-index: -1
```

Comment:
RIght, I was playing around with those before I used absolute positioning and did not realize I did not need them anymore. They are now removed.